pyflakes/checker.py

```diff
@@ -1678,6 +1678,10 @@ def CALL(self, node):
         ):
             self._handle_string_dot_format(node)
 
+        omit = []
+        annotated = []
+        not_annotated = []
+
         if (
             _is_typing(node.func, 'cast', self.scopeStack) and
```

Comment:
should we handle `cast` as well? I suspect that also has the same issue (since the first argument is a type)

Reply:
Cast isn't a named type so you can't use it within a subscript like `List[cast(A, 1)]`. The first argument will be either considered annotated if it's a string or it is be a type that

Reply:
here's a test case you can use which demonstrates my point:

```python
from typing import cast
from helpers import SomeGeneric

import queue

cast(SomeGeneric["queue.Queue[int]"], [])
```

Reply:
Ok I see, just have to make it for everything and not just strings I guess.